keyboards/geekboards/macropad_v2/keymaps/via/config.h

```diff
@@ -1,2 +1 @@
-#define DYNAMIC_KEYMAP_LAYER_COUNT 4
-#define VIA_QMK_RGBLIGHT_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```

Comment:
Please ensure all (changed) files have newlines at EOF, it muddies the diff a little.

Reply:
OK